rt/features/logging/src/main/java/org/apache/cxf/ext/logging/MaskSensitiveHelper.java

```diff
@@ -70,15 +71,19 @@ public String maskSensitiveElements(
             final Message message,
             final String originalLogString) {
         if (replacementsXML.isEmpty() && replacementsJSON.isEmpty()
-                || message == null
```

Comment:
Why this condition was removed? It is exactly what will be returned on line #80 but with unnecessary use of `Optional::ofNullable(message)`?

Reply:
The statement "contentType = Optional.ofNullable(message).map(m -> (String) m.get(Message.CONTENT_TYPE))" handles null message out of the box. Null check on top is redundant

Reply:
That is correct, so the question becomes: why do you need `Optional` at all? With the previous check in place, just:

```
final String contentType = (String) m.get(Message.CONTENT_TYPE));
if (contentType  == null) {
      return originalLogString;
}
```

It looks as straightforward as it gets (no need to create unnecessary instances).

Reply:
Matter of taste, for me Optional alternative is a bit compacter